govuk_frontend_jinja/templates.py

```diff
@@ -63,9 +64,48 @@ def njk_to_j2(template):
     # `length()`.
     template = template.replace(".length", " | length")
 
+    # see `indent_njk`
+    template = re.sub(re.escape("| indent") + r"\b", "| indent_njk", template)
+
     return template
 
 
+def indent_njk(
+    s, width=4, first=False, blank=False, indentfirst=None
+):
+    """A near-verbatim copy of jinja2's indent filter as of a2f5e2c7972c4d5148c1c75c724e24950d8605bc, including a fix
```

Comment:
I would suggest that the commentary should not be in the docstring.

```suggestion
    """Return a copy of the string with each line indented by 4 spaces."""
    
    # Copied from
    # https://github.com/pallets/jinja/blob/a2f5e2c7972c4d5148c1c75c724e24950d8605bc/jinja2/filters.py#L536-L580
    # to include an unreleased fix for https://github.com/pallets/jinja/pull/826
    # which causes the file upload component to escape HTML markup.
    # TODO: Remove once jinja2 2.11 is released and in use.
```